AK/Time.cpp

```diff
@@ -304,4 +308,160 @@ UnixDateTime UnixDateTime::now_coarse()
+                break;
+            case 'h':
+                TRY(builder.try_append(short_month_names[tm.tm_mon]));
+                break;
+            case 'H':
+                TRY(builder.try_appendff("{:02}", tm.tm_hour));
+                break;
+            case 'I': {
+                int display_hour = tm.tm_hour % 12;
+                if (display_hour == 0)
+                    display_hour = 12;
+                TRY(builder.try_appendff("{:02}", display_hour));
+                break;
+            }
+            case 'j':
+                TRY(builder.try_appendff("{:03}", tm.tm_yday + 1));
+                break;
+            case 'l': {
+                int display_hour = tm.tm_hour % 12;
+                if (display_hour == 0)
+                    display_hour = 12;
+                TRY(builder.try_appendff("{:2}", display_hour));
+                break;
+            }
+            case 'm':
+                TRY(builder.try_appendff("{:02}", tm.tm_mon + 1));
+                break;
+            case 'M':
+                TRY(builder.try_appendff("{:02}", tm.tm_min));
+                break;
+            case 'n':
+                TRY(builder.try_append('\n'));
+                break;
+            case 'p':
+                TRY(builder.try_append(tm.tm_hour < 12 ? "AM"sv : "PM"sv));
+                break;
+            case 'r': {
+                int display_hour = tm.tm_hour % 12;
+                if (display_hour == 0)
+                    display_hour = 12;
+                TRY(builder.try_appendff("{:02}:{:02}:{:02} {}", display_hour, tm.tm_min, tm.tm_sec, tm.tm_hour < 12 ? "AM" : "PM"));
+                break;
+            }
+            case 'R':
+                TRY(builder.try_appendff("{:02}:{:02}", tm.tm_hour, tm.tm_min));
+                break;
+            case 'S':
+                TRY(builder.try_appendff("{:02}", tm.tm_sec));
+                break;
+            case 't':
+                TRY(builder.try_append('\t'));
+                break;
+            case 'T':
+                TRY(builder.try_appendff("{:02}:{:02}:{:02}", tm.tm_hour, tm.tm_min, tm.tm_sec));
+                break;
+            case 'u':
+                TRY(builder.try_appendff("{}", tm.tm_wday ? tm.tm_wday : 7));
+                break;
+            case 'U': {
+                int const wday_of_year_beginning = (tm.tm_wday + 6 * tm.tm_yday) % 7;
+                int const week_number = (tm.tm_yday + wday_of_year_beginning) / 7;
+                TRY(builder.try_appendff("{:02}", week_number));
+                break;
+            }
+            case 'V': {
+                int const wday_of_year_beginning = (tm.tm_wday + 6 + 6 * tm.tm_yday) % 7;
+                int week_number = ((tm.tm_yday + wday_of_year_beginning) / 7) + 1;
+                if (wday_of_year_beginning > 3) {
+                    if (tm.tm_yday >= 7 - wday_of_year_beginning)
+                        --week_number;
+                    else {
```

Comment:
Please use braces for all parts of this `if` section ([style guideline](https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/CodingStyle.md#omission-of-curly-braces-from-statement-blocks)).

Reply:
Fixed